<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <widget class="QTextEdit" name="InputData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>111</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Print your substring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Find files</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="PathData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Pleas put your finding path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetPathButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set path</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="FilesOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>671</width>
      <height>261</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Finded files.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
